src/app/services/question.service.ts

```diff
@@ -43,6 +47,12 @@ export class QuestionService {
     return this.http.put<Question>(url, question, httpOptions);
   }
 
+  highlightResponse(question: Question): Observable<Question> {
```

Comment:
this is a generic update method.  Either change functionality to match the name or change the name to match the functionality